lib/cloudregister/smt.py

```diff
@@ -139,34 +142,31 @@ def is_equivalent(self, smt_server):
     # --------------------------------------------------------------------
     def is_responsive(self):
         """Check if the SMT server is responsive"""
-        # Per rfc3986 IPv6 addresses in a URI are enclosed in []
-        if self.get_ipv6():
-            health_url = 'https://[%s]/api/health/status' % self.get_ipv6()
-            cert_url = '%s://[%s]/smt.crt' % (self._protocol, self.get_ipv6())
-        else:
-            health_url = 'https://%s/api/health/status' % self.get_ipv4()
-            cert_url = '%s://%s/smt.crt' % (self._protocol, self.get_ipv4())
-
+        
         # We cannot know if the server cert has been imported into the
         # system cert hierarchy, nor do we know if the hostname is resolvable
         # or if the IP address is built into the cert. Since we only want
         # to know if the system is responsive we ignore cert validation
         # Using the IP address protects us from hostname spoofing
-        try:
-            response = requests.get(health_url, timeout=2, verify=False)
-            if response.status_code == 200:
-                status = response.json()
-                return status.get('state') == 'online'
-            elif response.status_code == 404:
-                # We are pointing to an SMT server, the health status API
-                # is not available. Download the cert to at least make sure
-                # Apache is responsive
-                cert_response = requests.get(cert_url, verify=False)
-                if cert_response and cert_response.status_code == 200:
-                    return True
-        except Exception:
-            # Something is wrong with the server
-            pass
+        for health_url, cert_url in self._check_urls.items():
```

Comment:
A bit nitpicking, maybe consider using the key only 
as the chances of pointing to a server with health status 
are quite high, then getting the `cert_url` with the key, looking like

```python
for health_url in self._check_urls.keys():
    ...
    elif response.status_code == 404:
        cert_url = self._check_urls.get(health_url)
        # or 
        cert_url = self._check_urls[health_url]
        # or no variable at all and
        cert_response = requests.get(
            self._check_urls.get(health_url) + cert_name, verify=False
        )
```
so the variable would get used near(er) the place it is declared